### ISSUE TITLE
Satellite admin web App: View project info

### ISSUE THREAD

**Issue by ifraixedes** [storj/storj#6476]
## Motivation / Why?

The new satellite administration Web App shows project information.

## What needs to be done?

- Define API endpoint to show project information through the API generator. 
- Implement the business logic of the endpoint.
- Wire the front-end with the API client generated by the API generator.
- Implement the needed front-end logic to accommodate the API client call.

Out of scope:
- To show project stats: number of segments, used storage, used bandwidth, etc.

## Acceptance criteria

- The new web App allows to click on the list of user's projects to navigate to a project information page.
- The new web App shows project information.

**Comment by ifraixedes** [comment]
Blocked by #6581

**Comment by ifraixedes** [comment]
For now, we are skipping the embedded structs without JSON tag, so I unblocked it

**Comment by ifraixedes** [comment]
It turns out that I implemented #6581 and I used it.

Commit ready for reviews https://review.dev.storj.io/c/storj/storj/+/11927

**Comment by ifraixedes** [comment]
The UI part is missing, I will post the commit link once it's done, however the API part is ready, as commented just above.

**Comment by ifraixedes** [comment]
Thanks to @cam-a the project view UI is implemented and merged.

**Comment by ifraixedes** [comment]
Deployed in release [v1.96.6](https://github.com/storj/storj/releases/tag/v1.96.6)
